src/main/java/io/mosip/mimoto/service/CredentialService.java

```diff
@@ -13,4 +13,5 @@
 public interface CredentialService {
     TokenResponseDTO getTokenResponse(Map<String, String> params, String issuerId) throws ApiNotAccessibleException, IOException;
     ByteArrayInputStream downloadCredentialAsPDF(String issuerId, String credentialType, TokenResponseDTO response) throws Exception;
+    Boolean verifyCredential(String credential);
```

Comment:
Lets throw the Exception in the method signature as well, Its imported but not used.

Reply:
added